src/Problems/PBO/f_ising_2D.hpp

```diff
@@ -37,7 +33,8 @@ class Ising_2D : public IOHprofiler_problem<int> {
     int n = x.size();
     int i,j,neig;
     int result= 0;
-    int neighbors[4];
+    // int neighbors[4];
+    int neighbors[2];
     int lattice_size = (int)sqrt((double)n);
```

Comment:
why converting a `size_t` variable to `double` then to `int`?

Reply:
It should use `static_cast`, and `size_t` should be prefferd to `int` for anything related to sizes (it's a common problem everywhere in IOHexperimenter, which raise a lot of downcasting warnings).